draft-ietf-gnap-core-protocol.md

```diff
@@ -5213,6 +5213,34 @@ method against the deployment capabilities of the client software and its
 environment. Due to the increased security, an interaction finish method should
 be employed whenever possible.
 
+## Session Management for Interaction Finish Methods {#security-sessions}
+
+When using an interaction finish method such as `redirect` or `push`, the client instance receives
+an unsolicited HTTP request from an unknown party, through the end-user's browser. The client
```

Comment:
Why would this request come from the end-user's browser in case of the push interaction finish mode? Isn't it rather the particularity of the push interaction finish mode that it is not the browser but the AS that finishes the interaction?